builtin/credential/approle/path_login.go

```diff
@@ -26,9 +26,17 @@ func pathLogin(b *backend) *framework.Path {
 				Description: "SecretID belong to the App role",
 			},
 		},
-		Callbacks: map[logical.Operation]framework.OperationFunc{
-			logical.UpdateOperation:         b.pathLoginUpdate,
-			logical.AliasLookaheadOperation: b.pathLoginUpdateAliasLookahead,
+		Operations: map[logical.Operation]framework.OperationHandler{
+			logical.UpdateOperation: &framework.PathOperation{
+				Callback:                    b.pathLoginUpdate,
+				ForwardPerformanceStandby:   true,
+				ForwardPerformanceSecondary: true,
+			},
+			logical.AliasLookaheadOperation: &framework.PathOperation{
+				Callback:                    b.pathLoginUpdateAliasLookahead,
+				ForwardPerformanceStandby:   true,
+				ForwardPerformanceSecondary: true,
```

Comment:
I don't think forwarding is needed for alias lookahead operations, but @ncabatoff or @briankassouf might know for sure.

Reply:
I hope it's not needed or we're going to have a lot more updates to make...

Reply:
Yeah, the way forwarding normally works (and this is the same whether using Callbacks or Operations) is that the perf standby tries to perform the request, it gets a read-only error internally, and the routing / generic handling code detects that and forwards the request instead of returning that error to the user.

The options you're using here are to force forwarding without doing that initial request.  This is typically because the operation doesn't perform any writes (so it can't use the mechanism I describe above), but we still want it to hit the active node for some reason.